### PR TITLE
Remove localStorage to store

### PR DESCRIPTION
**Pull request by sirromdev**

### CODE REVIEW

File: src/stores/Content.jsx [hunk 1]
```diff
@@ -64,19 +65,18 @@ export default class ContentStore {
...
-    return this.stabilityFeeMarkdown() && compiler(this.stabilityFeeMarkdown());
-  }
+  stabilityFeeContent = () => this.stabilityFeeMarkdown() && compiler(this.stabilityFeeMarkdown());
 
-  showStabilityFeeAlert = () => {
-    if (this.stabilityFeeMarkdown() === '') {
-      return false;
-    }
-    return true;
+  showStabilityFeeAlert = () =>
```
Comment (adrianleb) [inline src/stores/Content.jsx]: this is a boolean check no? `shouldShowStabilityFeeAlert`

File: src/stores/Content.jsx [hunk 2]
```diff
@@ -16,6 +16,7 @@ import contentTerms from "../assets/json/terms.json";
 export default class ContentStore {
   @observable content = { faq: {}, tooltips: {} }
   @observable contentLoaded = false
+  @observable showNotification = true;
```
Comment (adrianleb) [inline src/stores/Content.jsx]: hard to know without testing but can this maybe not be 

```js
@observable showNotication = !localStorage.getItem(`StabilityFeeChangeAlertClosed-${this.stabilityFeeMarkdown()}`)
```
?

File: src/components/StabilityFeeAlert.jsx [hunk 3]
```diff
@@ -9,22 +9,15 @@ import InlineNotification from "./InlineNotification";
...
-    this.state = { show: true }
-  }
-
   render() {
     return (
       this.props.content.showStabilityFeeAlert() &&
-        this.state.show &&
-        !localStorage.getItem(`StabilityFeeChangeAlertClosed-${this.props.content.stabilityFeeMarkdown()}`) &&
         this.props.network.isConnected &&
         this.props.network.defaultAccount &&
```
Comment (adrianleb) [inline src/components/StabilityFeeAlert.jsx]: still not sure why these two conditions are here :p

File: src/stores/Content.jsx (hunk 1 above)
Comment (sirromdev) [reply src/stores/Content.jsx]: fixed

File: src/stores/Content.jsx (hunk 2 above)
Comment (sirromdev) [reply src/stores/Content.jsx]: I tried this and it runs into issues until content is loaded. Moved to the `then()` of the axios request.

File: src/components/StabilityFeeAlert.jsx (hunk 3 above)
Comment (sirromdev) [reply src/components/StabilityFeeAlert.jsx]: https://github.com/makerdao/scd-cdp-portal/blob/2e3bdce90066346cf0dba563d71ef4424c0d4522/src/components/Home.jsx#L64

Should be the inverse of that to only render on the dashboard page as opposed to the landing

**Pull request merged**